src/TreeBasedInitialization.jl

```diff
@@ -12,59 +12,38 @@ function isCliqUpSolved(cliq::TreeClique)::Bool
 end
 
 
-
 """
     $SIGNATURES
 
 Return the most likely  ordering for initializing factor (assuming up solve
 sequence).
 
 Notes:
-- sorts id for increasing number of connected factors.
+- sorts id (label) for increasing number of connected factors using the clique subfg with messages already included.
 """
-function getCliqVarInitOrderUp(tree::BayesTree, cliq::TreeClique)
+function getCliqVarInitOrderUp(subfg::AbstractDFG)
   # rules to explore dimension from one to the other?
 
   # get all variable ids and number of associated factors
-  allids = getCliqAllVarIds(cliq)
-  nfcts = getCliqNumAssocFactorsPerVar(cliq)
+  B, varLabels, facLabels = getBiadjacencyMatrix(subfg)
```

Comment:
yep ,this is good -- later i want to use AMD, RCS, or CCOLAMD to work inside a clique just like over the whole graph but that is an improvement.  This is good for now.

For reference:
#980